src/libs/actions/Report.ts

```diff
@@ -3129,10 +3129,9 @@ function leaveRoom(reportID: string, isWorkspaceMemberLeavingWorkspaceRoom = fal
                               },
                           },
                       }
-                    : Object.keys(report).reduce<Record<string, null>>((acc, key) => {
-                          acc[key] = null;
-                          return acc;
-                      }, {}),
+                    : {
+                          reportName: report.reportName,
+                      },
```

Comment:
I think this is wrong, the onyx method is MERGE, so this won't remove the other report's keys from onyx. Can you do something like this instead?:

https://github.com/Expensify/App/issues/55676#issuecomment-2669622006